src/Symfony/Component/Notifier/Bridge/Nexmo/NexmoTransportFactory.php

```diff
@@ -32,6 +33,11 @@ public function create(Dsn $dsn): TransportInterface
         $apiKey = $this->getUser($dsn);
         $apiSecret = $this->getPassword($dsn);
         $from = $dsn->getOption('from');
+
+        if (!$from) {
+            throw new IncompleteDsnException('Missing from.', $dsn->getOriginalDsn());
+        }
```

Comment:
This should be considered a bugfix

Reply:
see https://github.com/symfony/symfony/pull/39406#discussion_r539307754